filterable-file-log/schema.lua

```diff
@@ -27,7 +27,7 @@ return {
   self_check = function(schema, config, dao, is_updating)
     for _, field in ipairs({"request_headers", "response_headers"}) do
       if config[field .. "_whitelist"] and config[field .. "_blacklist"] then
-        return false, Errors.schema "You cannot set both a whitelist and a blacklist for " .. field
+        return false
```

Comment:
Looking at the original source code linked in the comment at the top of the file, I think we can keep the error with code like this:

```
      return nil, string.format("You cannot set both a whitelist and a blacklist for: %s", field)
```

That seems consistent from 0.14.1 through 1.2.2:
- https://github.com/Kong/kong/blob/0.14.1/kong/plugins/file-log/schema.lua
- https://github.com/Kong/kong/blob/1.2.2/kong/plugins/file-log/schema.lua

I'm approving either way because upgrading to 1.2 is more important than the precise error text.